src/cwt/claim/ietf_token_status.rs

```diff
@@ -0,0 +1,79 @@
+//! Claims from the IETF Token Status List spec.
+
+use super::{Claim, Error as ClaimError, Key};
+use serde::{Deserialize, Serialize};
+use serde_cbor::Value;
+
+/// Status Claim for the Referenced Token, which describes
+/// its location in the Status List.
+#[derive(Clone, Debug, Deserialize, Serialize, PartialEq, PartialOrd, Eq, Ord, Hash)]
+pub struct Status {
+    pub status_list: StatusList,
+}
+
+#[derive(Clone, Debug, Deserialize, Serialize, PartialEq, PartialOrd, Eq, Ord, Hash)]
+pub struct StatusList {
```

Comment:
Rename to `StatusListInfo` to distinguish from the `StatusList` structure defined in section 4.2.
```suggestion
pub struct StatusListInfo {
```